sale_quotation_capable_vendor_rfq/models/product_product.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
+    def _get_matching_vendor_pricelists(self, quantity, output_time=0):
```

Comment:
`output_time` is a confusing name for this parameter. Should be something like `days_max`.
Let's also add a lower bottom and check it too. So there will be two of them:
`days_min`: minimum delivery lead time, days
`days_max`: maximum delivery lead time, days